service/matching/tasklist/task.go

```diff
@@ -57,6 +57,7 @@ type (
 		ResponseC                chan error // non-nil only where there is a caller waiting for response (sync-match)
 		BacklogCountHint         int64
 		ActivityTaskDispatchInfo *types.ActivityTaskDispatchInfo
+		PollLocalMatchLatencyMs  int64 // Poll local match latency per tasklist in ms
```

Comment:
it's the total poll latency not just local match cc @Shaddoll for confirmation.

Reply:
We have PollLocalMatchLatencyPerTaskList and PollForwardMatchLatencyPerTaskList, I only recorded the PollLocalMatchLatencyPerTaskList one here and set as PollLocalMatchLatencyMs. If we want to combine them, I will rename it to poll latency

Reply:
https://github.com/cadence-workflow/cadence/blob/master/service/matching/tasklist/matcher.go#L470 this has the different branches

Reply:
I think it should include not only local match. The latency should be the time spent waiting for a task.

Reply:
Yeah, Shijie has the same idea. Updated it to record after get response from tasklist manager tm.pollOrForward()